tools/topology/sof-cml-rt5682.m4

```diff
@@ -19,11 +19,17 @@ include(`platform/intel/'PLATFORM`.m4')
 
 DEBUG_START
 
+# if DMICPROC is not defined, default pipepline is volume
+ifdef(`HSMICPROC', , `define(HSMICPROC, volume)')
+ifdef(`HSEARPROC', , `define(HSEARPROC, volume)')
+ifdef(`DMICPROC', , `define(DMICPROC, passthrough)')
```

Comment:
Since this is direct attach DMIC the default should be eq-iir-volume and filter blob 40 Hz highpass for 48 kHz for dmic01 and 40 Hz highpass for 16 kHz for dmic16k. 

This reveals need for added DMIC16KPROC and DMIC16K_FILTER1 macros handling to get different filter blob for 16 kHz pipeline. The DMICPROC could be reused but for flexibility better to have them separated.